.env

```diff
@@ -10,3 +10,4 @@ AKASH_DEVCACHE_VERSIONS=${AKASH_DEVCACHE}/versions
 AKASH_DEVCACHE_NODE_MODULES=${AKASH_DEVCACHE}
 AKASH_DEVCACHE_NODE_BIN=${AKASH_DEVCACHE_NODE_MODULES}/node_modules/.bin
 AKASH_RUN=${AKASH_DEVCACHE}/run
+ASSUME_NO_MOVING_GC_UNSAFE_RISK_IT_WITH=go1.18
```

Comment:
What is this doing? Disabling some sort of check around pointers?

Reply:
thanks for catching. removing as it was only for testing